packages/flutter_blue_plus_platform_interface/lib/src/bluetooth_msgs.dart

```diff
@@ -877,10 +879,19 @@ class BmBondStateResponse {
 
 class BmCreateBondRequest {
   DeviceIdentifier remoteId;
+  Uint8List? pin;
 
   BmCreateBondRequest({
     required this.remoteId,
+    required this.pin,
```

Comment:
@chipweinberger could we remove the `required` here if possible so that this is not a breaking change?

```suggestion
    this.pin,
```

Reply:
its fine to break imo. The windows implementation is not started yet anyway.

Reply:
> whats the motivation for not breaking?

Not breaking the interface keeps it compatible with all the implementations without needing any changes.

> its fine to break imo.

No problem but ensure that you bump the major version when publishing the platform interface package.